types/store.go

```diff
@@ -105,12 +105,11 @@ type KVStore interface {
 
 	// Iterator over a domain of keys in ascending order. End is exclusive.
 	// Start must be less than end, or the Iterator is invalid.
-	// CONTRACT: No writes may happen within a domain while an iterator exists over it.
+	// To iterate over entire domain -> store.Iterator(nil, nil)
```

Comment:
Why remove these ? `CONTRACT` is how we indicate some essential requirement when using the method.

Reply:
ok, added it back and merged. thanks for working on this !!

Reply:
I believe that contract isn't necessary for the implementations of stores we have? At least that's my understanding from conversation with @sunnya97 and @rigelrozanski . Either way i think that should have been in separate PR.